### PR TITLE
Make README more friendly

### PR DESCRIPTION
**Pull request by nobrayner**
+ Adding definitions
+ Starting to reword other things

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,50 @@
...
+### `Find Target` :dart: `Communicate` :satellite: `Raid` :crossed_swords: `Push Code` :tada:
+
+One Raid at a time to maximize impact in short amount of time.
 
 ## Communication :smiley:
-1. Primarily www.kentcdodds.com/discord #oss-raid-group channel 
+
+We primarily communicate via [Discord][discord] in the `OSS-RAID-GROUP` category
+
+1. Primarily www.kentcdodds.com/discord OSS-RAID-GROUP Category, #:bee:-raid-general
```
Comment (Aprillion) [inline README.md:46]: hm, the 4th link to discord might be bit too many..

**Review by Aprillion — APPROVED** [review]

**Review by tigerabrodi — APPROVED** [review]
Looks good :fire:

File: README.md (hunk 1 above)
Comment (tigerabrodi) [reply README.md:46]: Yeah, agree with you there, we can just improve it later :tada:

Comment (nobrayner) [reply README.md:46]: Yep - I just changed this to a draft (didn't realise you could make drafts!)

I will be making more changes later, but it's always good to get more eyes early on 😄

File: README.md [hunk 2]
```diff
@@ -1,22 +1,68 @@
...
-3. Determine responsibilities then set to either Org member or Collab (reduced permissions)
+When participating in a Raid, use the following steps.
 
-## The Process find target -> communicate -> Raid -> Push Code :fire:
-1. One Target at a time to maximize impact in short amount of time.
+1. Raise an issue in the Raid repo with a one-liner title describing what you are working on - for example `Migrate page-header/breadcrumbs tests to testing-library`
+2. Fork the Raid Repo
+3. Create a branch named after your GitHub username - e.g. `nobrayner`
+4. Hack away :sunglasses::computer:
+5. Create a PR titled the same as your issue, and a desciption of `Fixes #{ISSUE_NO}` - [like this PR for example](https://github.com/kcdraidgroup/hospitalrun-frontend/pull/8)
```
Comment (nobrayner) [inline README.md:65]: @JacobMGEvans This is the process I have gleaned from the chat. I'd say it's good enough for now?

File: README.md [hunk 3]
```diff
@@ -1,22 +1,68 @@
...
+| Raid Group | The collective noun for all contributors to the Raid |
+| Raid Repo | The repository for the current Raid |
+
+## Storyline Quest (mission) :scroll:
+
+Objectives:
+
+* To make contributing to OSS enjoyable, engaging, and with a low barrier-to-entry for those who have never done it before
+* To gamify the process of mob programming OSS issues
+* To have a high impact in a small amount of time; specifically targetting low-medium contributor activity, medium-hgih usage of the project, with only a handful of maintainers
```
Comment (Aprillion) [inline README.md:22]: typo in `medium-hgih` ;)

File: README.md [hunk 4]
```diff
@@ -1,22 +1,68 @@
...
 - Themed from MMORPG gaming, gamifiying Mob programming OSS repos
 - Look for criteria in OSS repos 
   1. Low to medium contributor activity, only a handful of maintainers
   2. Medium to high usage of the OSS 
   3. Our impact can be potentially high; in a few days
+-->
+
+## Join Us! :smiling_face_with_three_hearts:
+
+Anoyone is welcome to participate in our Raids - no matter your experience level! Never contributed to an OSS project? No worries! We are all learning something or another, so come along for the ride!
```
Comment (merodiro) [inline README.md:34]: typo in Anoyone
```suggestion
Anyone is welcome to participate in our Raids - no matter your experience level! Never contributed to an OSS project? No worries! We are all learning something or another, so come along for the ride!
```

**Review by merodiro — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mpeyper) [reply README.md:65]: I'm not sure why you need a branch with your username if you're already working in a fork?  If anything I'd want to branch for my issue number so I can work on the next issue while another is up for review.

File: README.md [hunk 5]
```diff
@@ -1,22 +1,68 @@
...
+3. Create a branch named after your GitHub username - e.g. `nobrayner`
+4. Hack away :sunglasses::computer:
+5. Create a PR titled the same as your issue, and a desciption of `Fixes #{ISSUE_NO}` - [like this PR for example](https://github.com/kcdraidgroup/hospitalrun-frontend/pull/8)
 
-## Communication :smiley:
-1. Primarily www.kentcdodds.com/discord #oss-raid-group channel 
-2. Projects tab per Repo we are working on, to kanban who is working on what to prevent redudant work or stepping on toes.
 
-## Workflow :sunglasses:
+[discord]: https://kentcdodds.com/discord/
```
Comment (mpeyper) [inline README.md:68]: You can create links to specific channels in discord by using the "Create Invite" feature.  

image

Just a thought so new-comers don't get too lost on the big server (they might be new to discord too).

File: README.md (hunk 2 above)
Comment (JacobMGEvans) [reply README.md:65]: > I'm not sure why you need a branch with your username if you're already working in a fork? If anything I'd want to branch for my issue number so I can work on the next issue while another is up for review.

The issues are really your normal ticket, they are just there to track who is working on what, ultimately the PRs are just mild protection of the forked main/master but still hopefully keep the move fast capability.

File: README.md (hunk 5 above)
Comment (JacobMGEvans) [reply README.md:68]: Kent's Discord Bot I think prevents that, but we can try.

**Review by JacobMGEvans — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,22 +1,68 @@
...
 
-## Joining KCD Raid Group :smiling_face_with_three_hearts:
-1. Active Raid contributor
-2. Active Community member (KCD) 
-3. Determine responsibilities then set to either Org member or Collab (reduced permissions)
+When participating in a Raid, use the following steps.
 
-## The Process find target -> communicate -> Raid -> Push Code :fire:
-1. One Target at a time to maximize impact in short amount of time.
+1. Raise an issue in the Raid repo with a one-liner title describing what you are working on - for example `Migrate page-header/breadcrumbs tests to testing-library`
```
Comment (TheGallery) [inline README.md:61]: Can we please make clear that the raid repo is the kcdraidgroup fork and not the original repository?

Comment (tigerabrodi) [reply README.md:61]: Sounds like a good idea :tada:

**Review by merodiro — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (nobrayner) [reply README.md:68]: I was debating doing a direct link... We can give it a go 👍

Comment (nobrayner) [reply README.md]: Done

File: README.md (hunk 6 above)
Comment (nobrayner) [reply README.md:61]: @TheGallery - It doesn't actually relate to this line, but I updated the definitions table to be explicit about it being the repo in the kcdraidgroup org.

**Comment by nobrayner** [comment]
@JacobMGEvans If you are happy with this, I will merge?

**Pull request merged**